packages/common/core-utils/package.json

```diff
@@ -52,9 +84,11 @@
 		"test": "npm run test:mocha",
 		"test:benchmark:report": "mocha --node-option unhandled-rejections=strict,expose-gc --exit --perfMode --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha",
+		"test:mocha": "npm run test:mocha:cjs",
+		"test:mocha:cjs": "mocha  --recursive \"dist/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:esm": "mocha  --recursive \"lib/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
```

Comment:
This looks like mocha is configured differently than just from command line. Look at the configuration of packages/dds/tree for a comparison.

Reply:
Good catch.  This package was setting the file spec in '.mocharc.cjs'.  I've removed that to make it like the others.